lib/Conversion/QCToCatalystQuantum/QCToCatalystQuantum.cpp

```
@@ -0,0 +1,329 @@
+}
+
+/// Helper function to extract operands and control info - for more complex
+/// cases
+template <typename OpAdaptor> struct ExtractedOperands {
+  ValueRange inQubits;
+  ControlInfo ctrlInfo;
+};
+
+template <typename OpAdaptor>
+ExtractedOperands<OpAdaptor>
+extractOperands(OpAdaptor adaptor, ConversionPatternRewriter& rewriter,
```

Cppcheck (2.20.0)

[style] The function 'extractOperands' is never used.

(unusedFunction)